package.json

```diff
@@ -31,9 +31,8 @@
     "nodemon": "nodemon ./app.js -e html,js --ignore built --ignore frontend",
     "nodemon:debug": "nodemon --inspect-brk=1338 ./app.js -e html,js --ignore built --ignore frontend",
     "lint": "eslint --ignore-path .gitignore .",
-    "pretest": "npm run lint",
```

Comment:
What happened to pretest ? We don't use it anywhere?

Reply:
I didn't like that linting happened before the tests when you ran npm test so I move npm run test **into** the test script. Basically i want to focus on my tests passing not on semi colons first. I want linting throwing errors _after_ everything on tests is passing. 

So now the tests run, and then the linting runs.

Happy to undo if wanted

Reply:
it makes a lot of sense. Happy to delete it

Reply:
> it makes a lot of sense. Happy to delete it

@HerveH44 I did not quite get your comment, can you clarify please?

Do you think we don't need the lint run here at all?
Because this change only moved and combined it with the line below, nothing is deleted. It still runs with every `npm test` as better seen [here](https://github.com/dr4fters/dr4ft/pull/1419/files/e50371518577c49b46cbd7406f3e8f89982f3d16#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R34).

So if there is a linting error, 3 CI builds will fail. The `test 12`, `test 14` and dedicated `eslint` one [which only runs lint](https://github.com/dr4fters/dr4ft/blob/master/.github/workflows/tests.yml#L87).

Not sure about the local dev process, but I assume we can remove it from test because in a PR's the linting is still checked and one can easily run `npm run lint` locally?